packages/react-shortcuts/src/ShortcutProvider/ShortcutProvider.tsx

```diff
@@ -9,8 +9,9 @@ export interface Props {
   children?: React.ReactNode;
 }
 
-export const {Provider, Consumer} = React.createContext<Context>({});
+export const ShortcutContext = React.createContext<Context>({});
 
+export const {Consumer, Provider} = ShortcutContext;
```

Comment:
Figured we should still export these individually even though they are included in the above line.

Reply:
We'll want to cut a new major version (if we rewrite the component with a hook), so I think we should stop exporting these separately (also, if we could make it so context isn't a big old object, that would be 💯)